Src/Newtonsoft.Json.UnityConverters.Tests/ConvertingUnityTypes/BoundsTests.cs

```diff
@@ -0,0 +1,26 @@
+﻿using UnityEngine;
+
+namespace Newtonsoft.Json.UnityConverters.Tests.ConvertingUnityTypes
+{
+    public class BoundsTests : TypeTester<Bounds>
+    {
+        public static (Bounds deserialized, string serialized)[] representations = new[] {
+            (new Bounds(new Vector3(1, 2, 3), new Vector3(4, 5, 6)), @"{""xMin"":1,""yMin"":2,""zMin"":3,""sizeX"":4,""sizeY"":5,""sizeZ"":6}")
+        };
+
+        protected override void ConfigureSettings(JsonSerializerSettings settings)
+        {
+        }
+    }
+
+    public class BoundsIntTests : TypeTester<BoundsInt>
+    {
+        public static (BoundsInt deserialized, string serialized)[] representations = new[] {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Change the visibility of 'representations' or make it 'const' or 'readonly'.](https://app.codacy.com/manual/jilleJr/Newtonsoft.Json-for-Unity.Converters/pullRequest?prid=5190738)